executing/executing.py

```diff
@@ -530,6 +539,110 @@ def visit_ClassDef(self, node):
         self.stack.pop()
 
 
+class PositionNodeFinder(object):
+    def __init__(self, frame, stmts, tree, lasti, source):
+        # we can use co_positions() since 3.11, which has fewer limitations
+
+        bc_list = list(dis.Bytecode(frame.f_code, show_caches=True))
```

Comment:
Is there a difference between this and `get_instructions`? If not, I think I prefer to consistently use `get_instructions` for simplicity.

Reply:
no, I change it.